Compiler warning on missing prototypes (Mac)

./Modules/readline.c:1018:16: warning: a function declaration without a prototype is deprecated in all versions of C [-Wstrict-prototypes]
on_startup_hook()
               ^
                void
./Modules/readline.c:1033:18: warning: a function declaration without a prototype is deprecated in all versions of C [-Wstrict-prototypes]
on_pre_input_hook()
                 ^
                  void
2 warnings generated.


<!-- gh-linked-prs -->
### Linked PRs
* gh-105241
<!-- /gh-linked-prs -->

--- a/Modules/readline.c
+++ b/Modules/readline.c
@@ -1015,7 +1015,7 @@ static int
 #if defined(_RL_FUNCTION_TYPEDEF)
 on_startup_hook(void)
 #else
-on_startup_hook()
+on_startup_hook(void)
 #endif
 {
     int r;
@@ -1030,7 +1030,7 @@ static int
 #if defined(_RL_FUNCTION_TYPEDEF)
 on_pre_input_hook(void)
 #else
-on_pre_input_hook()
+on_pre_input_hook(void)
 #endif
 {
     int r;

